modules/core/shared/src/main/scala/lucuma/core/model/EmissionLine.scala

```diff
@@ -4,21 +4,24 @@
 package lucuma.core.model
 
 import cats.Eq
-import cats.implicits._
+import cats.data.NonEmptyMap
+import cats.implicits.*
 import coulomb.*
 import coulomb.ops.algebra.cats.all.given
 import coulomb.syntax.*
-import eu.timepit.refined.cats._
+import eu.timepit.refined.cats.*
 import eu.timepit.refined.types.numeric.PosBigDecimal
-import lucuma.core.math.BrightnessUnits._
-import lucuma.core.math.dimensional._
-import lucuma.core.math.units._
+import lucuma.core.math.BrightnessUnits.*
+import lucuma.core.math.dimensional.*
+import lucuma.core.math.units.*
 import monocle.Focus
 import monocle.Lens
 
+import java.time.Instant
+
 final case class EmissionLine[T](
   lineWidth: Quantity[PosBigDecimal, KilometersPerSecond],
-  lineFlux:  Measure[PosBigDecimal] Of LineFlux[T]
+  lineFlux:  EmissionLine.LineFluxOverTime[T]
```

Comment:
I don't know anything about this unfortunately.  It makes sense that the "line flux" would vary but are we sure that it does?

Reply:
I'll verify, my input from science is that emission lines can also vary over time